GW2EIEvtcParser/ParsedData/FightData.cs

```diff
@@ -13,6 +13,7 @@ public class FightData
         private List<PhaseData> _phases = new List<PhaseData>();
         private List<PhaseData> _nonDummyPhases = new List<PhaseData>();
         public int TriggerID { get; }
+        public int EvtcVersion { get; }
```

Comment:
That attribute is already on LogData, I don't think it is a good idea to duplicate it here. FightData is also not the best place to store evtc related meta data.
You can simply modify the signature of GetFightOffset and give evtcVersion to it, the method is called in EvtcParser.cs where the value also exists.